Dockerfile

```
@@ -1,19 +1,37 @@
+ARG TARGETARCH
+RUN echo "Target architecture is: $TARGETARCH"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     jq \
+    xz-utils \
+    tar \
     sqlite3 \
     && rm -rf /var/lib/apt/lists/*
 
 COPY requirements.txt .
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)